install/infra/modules/eks/kubernetes.tf

```
@@ -149,7 +149,11 @@ module "eks" {
         device_name = "/dev/sda1"
 
         ebs = [{
-          volume_size = 300
+          volume_size           = 360
```

We've changed our thinking here, see PR: https://github.com/gitpod-io/website/pull/2630

```suggestion
          volume_size           = 512
```